client/src/components/CardWrapper/CardWrapper.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Box, Paper } from '@mui/material';
+
+interface CardWrapperProps {
+  children: React.ReactNode;
+}
+
+const CardWrapper: React.FC<CardWrapperProps> = ({ children }) => {
```

Comment:
We can use the MUI Card API and assign a class to it with the below styles, I don't think we need to make a separate `CardWrapper` here :)